boards/arm/mimxrt1060_evk/mimxrt1060_evk-pinctrl.dtsi

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2021, NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/mcux_rt-pinctrl.h>
+
+&pinctrl {
+	sai1_default: sai1_default {
+		IOMUXC_GPIO_AD_B1_09_SAI1_MCLK {
+			pinmux = <0x401F8120 0x3 0x401F858C 0x1 0x401F8310 0x1>;
```

Comment:
Maybe move the `inputOnfield` parameter in the `pinmux` property to be a configurable value instead of part of the pin definition? My reasoning here is that the the `inputOnfield` property is usually given as an argument to `IOMUXC_SetPinMux`, where as the rest of the constants defined here are not

Reply:
I assume that a general guide is that using dts to confingure what is required. if we make it as an argument, so we need put this value to driver?